frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/top-consumers-card/top-consumers-card.tsx

```diff
@@ -30,32 +33,71 @@ const TopConsumerResourceValueMapping = {
   Pods: 'pod',
 };
 
-const metricTypes = _.keys(TopConsumerResourceValue);
-const sortByTypes = _.keys(TopConsumerSortByValue);
+const getTopConsumersQueriesMap = (
+  pluginItem: DashboardStorageExtension,
+  topConsumers: TopConsumersQueries,
+  category: string,
+): TopConsumersQueries => {
```

Comment:
Nit: this function mutates `topConsumers` parameter so I'd prefer to use a more appropriate name (e.g. `updateTopConsumersQueries`) and have its return type as `void`.

I will post a follow-up PR to address this.